bundles/org.openhab.binding.jeelink/src/main/java/org/openhab/binding/jeelink/internal/connection/AbstractJeeLinkConnection.java

```diff
@@ -116,4 +116,4 @@ public static JeeLinkConnection createFor(JeeLinkConfig config, ScheduledExecuto
 
         return connection;
     }
-}
+}
```

Comment:
Some of our build tools don't like EOF without a newline. So add a newline here.

Reply:
Done.